src/renderers/webgl/WebGLAttributes.js

```diff
@@ -19,7 +19,7 @@ function WebGLAttributes( gl, capabilities ) {
 
 		let type;
 
-		if ( array instanceof Float32Array ) {
+		if ( array instanceof Float32Array || array instanceof Float64Array ) {
```

Comment:
Unfortunately, this does not solve your issues since `Float64Array` can't be used as an input for `gl.Float`. Test your change with the following code:
```js
import * as THREE from 'three';

let renderer, scene, camera;

init();
render();

function init() {

	renderer = new THREE.WebGLRenderer( { antialias: true } );
	renderer.setSize( window.innerWidth, window.innerHeight );
	renderer.setPixelRatio( window.devicePixelRatio );
	document.body.appendChild( renderer.domElement );

	scene = new THREE.Scene();

	camera = new THREE.PerspectiveCamera( 40, window.innerWidth / window.innerHeight, 1, 10000 );
	camera.position.set( 0, 0, 2 );

	const geometry = new THREE.BufferGeometry();

	const data = new Float64Array( 9 );

	data[ 0 ] = 0.5;
	data[ 1 ] = - 0.5;
	data[ 2 ] = 0;
	data[ 3 ] = 0;
	data[ 4 ] = 0.5;
	data[ 5 ] = 0;
	data[ 6 ] = - 0.5;
	data[ 7 ] = - 0.5;
	data[ 8 ] = 0;

	const attribute = new THREE.BufferAttribute( data, 3 );
	geometry.setAttribute( 'position', attribute );

	const material = new THREE.MeshBasicMaterial();

	const mesh = new THREE.Mesh( geometry, material );
	scene.add( mesh );

}

function render() {

	renderer.render( scene, camera );

}
```

Reply:
Yes, it runs normally when using Float32Array. Because I saw Float64Array in BufferAttribute Types, I thought the judgment logic was missing.